actix-files/src/lib.rs

```diff
@@ -1222,7 +1222,10 @@ mod tests {
         );
 
         let bytes = test::read_body(resp).await;
+        #[cfg(unix)]
         assert!(format!("{:?}", bytes).contains("/tests/test.png"));
+        #[cfg(windows)]
+        assert!(format!("{:?}", bytes).contains("/tests\\\\test.png"));
```

Comment:
I think, this is wrong. We should always return `/` even for windows

Reply:
Added a fix, could you review it?